homeassistant/components/bring/quality_scale.yaml

```diff
@@ -7,7 +7,7 @@ rules:
   brands: done
   common-modules: done
   config-flow-test-coverage: done
-  config-flow: done
+  config-flow: todo
```

Comment:
Or you could just add the missing translation...

Reply:
Don't have the time right now. If this can wait i'll fix the translations. If it is urgent, merge this and I can work on it later without pressure ^^